router/src/infer.rs

```diff
@@ -666,3 +693,20 @@ impl InferError {
         }
     }
 }
+
+#[derive(Debug, Error)]
+pub enum ChatTemplateError {
+    #[error("Template error: {0}")]
+    TemplateError(#[from] minijinja::Error),
+    #[error("Template not found")]
+    TemplateNotFound,
+}
+
+impl ChatTemplateError {
+    pub(crate) fn error_type(&self) -> &str {
```

Comment:
Why do we need this ?

Reply:
thanks for catching, `ChatTemplateError` is unnecessary and has been removed in favor of adding a `InferError::TemplateError(_)`